### PR TITLE
Ensure Jest configuration for testing

### PR DESCRIPTION
**Pull request by andersfischernielsen**
This will:
- Remove invalid test cases
- Add tests for `src/app.ts`

### CODE REVIEW

File: src/templates.ts [hunk 1]
```diff
@@ -78,6 +78,13 @@ export const downloadTemplates =
     })) as OctokitResponse<ArrayBuffer>
     console.debug('Fetched release contents.')
 
+    if (typeof contents === 'string') {
+      return {
+        contents: Uint8Array.from(JSON.parse(contents)).buffer,
+        version: release.tag_name,
+      }
+    }
+
```
Comment (dpotyralski) [inline src/templates.ts:87]: Why did you decide to add this part? Could you help me understand? In what cases we can get String from `octokit.repos.downloadZipballArchive`?

Comment (andersfischernielsen) [reply src/templates.ts:87]: Well we wouldn't - but Nock doesn't allow mocking a HTTP request that returns an ArrayBuffer 🤔 I'll try to have a rethink on how to do that

Comment (andersfischernielsen) [reply src/templates.ts]: Fixed!

**Review by dpotyralski — APPROVED** [review]
LGTM! 🍭

**Pull request merged**